### ISSUE TITLE
Very bad results

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/axondeepseg#38]
**Version:**
6c7d20a2fa3a19b323b3c1e87ded2c943a7e344e

**Data:**
Source: https://osf.io/28k4m/
![does_kelm_072214_021](https://user-images.githubusercontent.com/2482071/34977116-ac83d6ce-fadc-11e7-85b3-53941d549370.png)

**Syntax:**
~~~
axondeepseg -t TEM -p Does_Kelm_072214_021.png -v 2 -s 0.004
~~~

**Results:**
![_segmenteddoes_kelm_072214_021](https://user-images.githubusercontent.com/2482071/34977136-b818dafc-fadc-11e7-9325-3853a51e76d2.png)

**Comment by perone** [comment]
I wonder if this isn't because of the png input (that can have an alpha channel or something like that), but the weird part is that there are some spikes of predictions around the borders.

**Comment by alzaia** [comment]
This was definitely because of the pre-processing that was missing. Tested same sample with last version. The results are now OK:
![34977116-ac83d6ce-fadc-11e7-85b3-53941d549370_segmented](https://user-images.githubusercontent.com/14980394/36710456-1ff751b4-1b4c-11e8-974c-82a45d6016ae.png)

**Comment by jcohenadad** [comment]
We can indeed close. Issue is solved in 759a45f581d2fd5ddd47ce845059f607466d3956.

@alzaia please use commit numbers instead of "last version", which we cannot track down.
